README.md

```diff
@@ -1,10 +1,22 @@
 ### Metabolic Atlas Map Viewer 3D
 
-This is a 3D graph viewer to visualize the metabolic atlas compound graph (metabolic network). See the [wiki](../../wiki) for more details.
+This is a 3D graph viewer to visualize the metabolic atlas compound graph (metabolic network), which is used as a self-contained feature of [Metabolic Atlas](https://metabolicatlas.org). The purpose of this library is to delvier the right technical solution that allows Metabolic Atlas to provide users with a useful experience while browsing and modifying a very large graph, with minimal knowledge required. The solution should be easy to work with and easy to maintain by programmers. The expectation is that most of the users of Metabolic Atlas will be biologists, followed by bioinformaticians; this means the 3D viewer should be minimalistic and intuitive. 
 
-Install dependencies with `npm install`, then run `npm run dev` to start the
-development server, or `npm run prepare` to build and minify the
-current version of the viewer.
+Below is a generic metabolic network example. Other graph structures are possible: no genes, duplicated metabolites.
+```
+Genes - Reaction - Metabolite - Reaction - Metabolite          
```

Comment:
Should `Genes` be singular?

```suggestion
Gene - Reaction - Metabolite - Reaction - Metabolite
```

Reply:
I guess it could. I wanted to communicate that there are multiple genes connected to a reaction (thinking of how things look like in the 3D viewer).

Reply:
That makes sense. Do we want to have the same for metabolites?

Reply:
Yeah that would make sense to me, let's what others think:
```suggestion
Genes - Reaction - Metabolites - Reaction - Metabolite          
```

Reply:
I think it's clearer if we always go for singular. The fact that a reaction can be connected to multiple metabolites is alreay conveyed (`Metabolite - Reaction - Metabolite`), so maybe we can do the same for genes, eg
```
           Gene
            | 
Gene - Reaction 
```

Reply:
👍🏻